src/odemis/acq/stream/_helper.py

```diff
@@ -586,12 +586,19 @@ class ARSettingsStream(CCDSettingsStream):
 
     See StaticARStream for displaying a stream with polar projection.
     """
-    def __init__(self, name, detector, dataflow, emitter, **kwargs):
+    def __init__(self, name, detector, dataflow, emitter, analyzer=None, **kwargs):
         if "acq_type" not in kwargs:
             kwargs["acq_type"] = model.MD_AT_AR
+
         super(ARSettingsStream, self).__init__(name, detector, dataflow, emitter, **kwargs)
         # For SPARC: typical user wants density much lower than SEM
         self.pixelSize.value *= 30
+        self.analyzer = analyzer
+        if analyzer:
+            # hardcode the 6 polarization positions plus the option to record all 6 positions sequentially
+            self.polarization = model.VAEnumerated({},
+                                         choices={POL_6POS, "pass-through",
+                                         'horizontal', 'vertical', 'posdiag', 'negdiag', 'rhc', 'lhc'})
```

Comment:
Let's put the list of positions as either one constant, or each position as a separate position.